platform/android/app/build.gradle

```diff
@@ -220,71 +220,20 @@ task copyThirdPartyLicenses(type: Copy) {
     into "build/generated-assets/3rd-party-licenses"
 }
 
-task installAppOnDevice {
```

Comment:
Not very sure, just checking - after removing this, are instrumentation tests work properly?

Reply:
Yes - this step always bugged me a bit, I see no need to somehow manually install the app on the phone, that's being done for us...